include/xeus-python/xdebugger.hpp

```diff
@@ -71,7 +71,7 @@ namespace xpyt
         std::string m_debugpy_port;
         nl::json m_debugger_config;
         py::object m_pydebugger;
-        std::string m_debugpy_version;
+        bool m_version_lt_1_6_5;
```

Comment:
I think that `m_copy_to_globals_available` would be more explicit, the fact that it is available onl after version 1.6.5 is an implementation detail.

Reply:
right, thanks.